x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -382,19 +382,21 @@ export function LayerPanel(
 
             if (!isEmptyLayer) {
               if (group.requiredMinDimensionCount) {
-                errorText = i18n.translate(
-                  'xpack.lens.editorFrame.requiresTwoOrMoreFieldsWarningLabel',
-                  {
-                    defaultMessage: 'Requires {requiredMinDimensionCount} fields',
-                    values: {
-                      requiredMinDimensionCount: group.requiredMinDimensionCount,
-                    },
-                  }
-                );
-              } else if (group.required && group.accessors.length === 0) {
-                errorText = i18n.translate('xpack.lens.editorFrame.requiresFieldWarningLabel', {
-                  defaultMessage: 'Requires field',
-                });
+                if (group.requiredMinDimensionCount > 1) {
```

Comment:
I have no clue how it worked before when we had this case, but when I changed `requiredMinDimensionCount` for metrics, table, I still get the error when having 2 metrics:
<img width="313" alt="Screenshot 2022-09-28 at 13 13 59" src="https://user-images.githubusercontent.com/4283304/192765360-eac22ccd-39aa-4cd4-9aa1-60e5e64a7203.png">

```suggestion
                if (group.requiredMinDimensionCount > 1 && group.accessors.length >= 1 ) {
```